provider/cmd/pulumi-resource-google-native/metadata.json

```diff
@@ -82405,8 +82405,7 @@
                 "name": "name",
                 "organizationsId": "organizationId",
                 "type": "type"
-            },
-            "autoNamePattern": "{name}"
+            }
```

Comment:
Curious why this shows up?

Reply:
The name is actually required in this case, but that was missed because the property description was missing. I think the auto-naming logic still needs work, but this is consistent with the current behavior.